### ISSUE TITLE
Element returns aria-hidden=true property only during TestCafe script.

### ISSUE THREAD

**Issue by harry7771** [DevExpress/testcafe#8064]
### What is your Scenario?

Click a dropdown in my company's Workday app via TC script.

### What is the Current behavior?

A 'button' element returns a property of 'display: none' or aria-hidden=true during a run of the TC script, but the element does not return those properties when the same workflow is run manually. My script is therefore unable to click the screen element.
Here is the TC error:
The action target (<button data-automation-activebutton="true" title="Edit" aria-hidden="true" type="button">...</button>) is invisible.
      It descends from an element that has the 'display: none' property (<div data-automation-id="dropDownCommandButton" aria-hidden="true" >...</div>).

### What is the Expected behavior?

I would expect TC to return the same element properties during the automated script as when it is run manually.

### What is the public URL of the test page? (attach your complete example)

Unfortunately there is no public access to this site.

### What is your TestCafe test code?

const editStatementDropdown = Selector('div[title="Statements"]').find('button[title="Edit"]');

### Your complete configuration file

no config file in use

### Your complete test report

A 'button' element returns a property of 'display: none' or aria-hidden=true during a run of the TC script, but the element does not return those properties when the same workflow is run manually. My script is therefore unable to click the screen element.
Here is the TC error:
The action target (<button data-automation-activebutton="true" title="Edit" aria-hidden="true" type="button">...</button>) is invisible.
      It descends from an element that has the 'display: none' property (<div data-automation-id="dropDownCommandButton" aria-hidden="true" >...</div>).

### Screenshots

![image](https://github.com/DevExpress/testcafe/assets/31902350/e58124b5-c0ad-4230-bf66-b1aee752c88a)

### Steps to Reproduce

1.
2.
3.

### TestCafe version

3.1.0

### Node.js version

18.17.1

### Command-line arguments

testcafe chrome demo.test.js -e

### Browser name(s) and version(s)

latest of Chrome and Firefox

**Comment by harry7771** [comment]
Fixed!
Simply added the following to the existing selector:  .filterVisible()

**Comment by harry7771** [comment]
Fixed!
Simply added the following to the existing selector:  .filterVisible()
